<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1368</width>
    <height>908</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="main_vertical_splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="main_horizontal_splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QGroupBox" name="flows_groupBox">
         <property name="title">
          <string>Flows</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </widget>
        <widget class="QGroupBox" name="nodes_groupBox">
         <property name="title">
          <string>Nodes</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </widget>
       </widget>
       <widget class="QTabWidget" name="flows_tab_widget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Main</string>
         </attribute>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="console_placeholder_widget" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1368</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuScripts">
     <property name="title">
      <string>Scripts</string>
     </property>
     <addaction name="actionNew_Flow"/>
     <addaction name="actionRename_Flow"/>
     <addaction name="actionDelete_Flow"/>
    </widget>
    <addaction name="actionImport_Nodes"/>
    <addaction name="actionImport_Example_Nodes"/>
    <addaction name="actionSave_Project"/>
    <addaction name="menuScripts"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuFlow_Design_Style">
     <property name="title">
      <string>Flow Theme</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuSave_Picture">
     <property name="title">
      <string>Save Picture</string>
     </property>
     <addaction name="actionSave_Pic_Viewport"/>
     <addaction name="actionSave_Pic_Whole_Scene_scaled"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuFlow_Design_Style"/>
    <addaction name="menuSave_Picture"/>
   </widget>
   <widget class="QMenu" name="menuDebugging">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuInfo_Messages">
     <property name="title">
      <string>Info Messages</string>
     </property>
     <addaction name="actionEnableInfoMessages"/>
     <addaction name="actionDisableInfoMessages"/>
    </widget>
    <addaction name="menuInfo_Messages"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuDebugging"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport_Nodes">
   <property name="text">
    <string>Import Nodes</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionDesignDark_Std">
   <property name="text">
    <string>Dark Std</string>
   </property>
  </action>
  <action name="actionDesignDark_Tron">
   <property name="text">
    <string>Dark Tron</string>
   </property>
  </action>
  <action name="actionEnableInfoMessages">
   <property name="text">
    <string>Enable</string>
   </property>
  </action>
  <action name="actionDisableInfoMessages">
   <property name="text">
    <string>Disable</string>
   </property>
  </action>
  <action name="actionSave_Pic_Viewport">
   <property name="text">
    <string>Save Pic - Viewport</string>
   </property>
   <property name="toolTip">
    <string>Save a picture of the current scene's viewport.
This will save exactly what you see in the same resolution.</string>
   </property>
  </action>
  <action name="actionSave_Pic_Whole_Scene_scaled">
   <property name="text">
    <string>Save Pic - Whole Scene (scaled)</string>
   </property>
   <property name="toolTip">
    <string>Saves a picture of the whole current scene. 
The more you zoomed in, the sharper the picture.
This will take a few seconds.</string>
   </property>
  </action>
  <action name="actionNew_Flow">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionRename_Flow">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionDelete_Flow">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionImport_Example_Nodes">
   <property name="text">
    <string>Import Example Nodes</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
